ivy/functional/ivy/experimental/manipulation.py

```diff
@@ -1593,3 +1594,41 @@ def expand(
         Output Array
     """
     return ivy.current_backend(x).expand(x, shape, out=out)
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_function
+def concat_from_sequence(
+    input_sequence: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    new_axis: int = 0,
+    axis: int = 0,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
+    Concatenates arrays if axis = 0 and stacks arrays if axis = 1.
+
+    Parameters
+    ----------
+    input_sequence
+        Array input.
+    new_axis
+        Decides wether to stack or not.
```

Comment:
"Insert and concatenate on a new axis or not, default 0 means do not insert new axis."